docs/updates.thunderbird.net.md

```diff
@@ -1,57 +1,344 @@
+
+```jinja
+{{ high_res_img('thunderbird/appeal/dec24/forest-roc.png', {'alt': _('')}, alt_formats=('webp', 'avif')) }}
+```
+
+### Donation Button
+
+A reusable donation button is defined in `includes/macros/donate-button.html`.
+
+There are three steps to using it:
+
+1. Import the macro
+2. Define the variables for URL generation
+3. Call the macro
+
+#### Import the macro
+
+First add the following to your template, after the `extends`:
+
+```jinja
+{% from 'includes/macros/donate-button.html' import donate_button with context %}
+```
+
+#### Define the variables for URL generation
+
+Near the top of your Jinja template, declare the following variables:
+
+```jinja
+{% set fru_form_id = fru_form_id|default('eoy2024') %}
+{% set utm_campaign = utm_campaign|default('dec24_appeal') %}
+{% set donation_base_url = donation_base_url|default(url('updates.128.appeal.dec24.donate')) %}
+```
+
+Confirm the values for `fru_form_id` and `utm_campaign` with the Marketing and Communications team.
+
+The `donation_base_url` variable comes from the `URL_MAPPINGS` dict in `settings.py`. You'll need to add to this dict each time you use the donate_button macro.
+
+Here is the entry for the December 2024 appeal:
+
+```py
+'updates.128.appeal.dec24.donate': '/thunderbird/128.0/dec24/donate/',
+```
+
+***Make sure to restart the build tool after editing the `settings.py` file.***
+
+#### Call the macro
+
+Finally, call the macro where you'd like to display it:
+
+```jinja
+<section id="donate-button-container">
+  {{ donate_button(form_id=fru_form_id, campaign=utm_campaign, base_url=donation_base_url) }}
+</section>
+```
+
+After restarting the build tool and refreshing your browser, you should see the donation button on the page.
+
+### Localization
+
+All text in the Jinja template should be translated into the user's local language.
+
+#### Specifying translatable strings
+
+There are two ways to specify that a string should be translated.
+You can wrap the string in one of the following:
+
+- The `_()` function is used for shorter pieces of text.
+- A `{% trans %}` block wraps longer strings.
+
+Use `_()` for simple strings:
+
+ ```jinja
+ <p class="closing-text">{{ _('The Thunderbird Team') }}</p>
+ ```
+
+HTML tags are allowed inside a translatable string:
+
+```jinja
+<h1>
+    {{ _('Help Keep <span>Thunderbird Alive</span>') }}
+</h1>
+```
+When possible, avoid putting HTML tags inside a translatable string. One of our volunteer translators may accidentally change the tags, causing an error when building the site.
+
+`aria-label` (and `alt` text) should be translated:
+
+```jinja
+<h1 id="appeal-heading" aria-label="{{ _('Help Keep Thunderbird Alive!') }}">
+  {{ _('Help Keep <span>Thunderbird Alive</span>') }}
+</h1>
+```
+
+The December 2024 appeal does not use `{% trans %}` blocks, but here is an example from the main `index.html`:
+
+```jinja
+{% trans %}
```

Comment:
We'll need to add trimmed to trans otherwise it gets funky on pontoon.
```suggestion
{% trans trimmed %}
```

We could also add a section about using variables in trans blocks, but that's technically covered by the jinja guide.

Reply:
Do we usually do this for appeals pages? Happy to add a section (or at least link to the jinja docs that mention this)